src/components/ui/Breadcrumb/breadcrumb.scss

```diff
@@ -3,36 +3,61 @@
 [data-store-breadcrumb] {
   width: 100%;
 
+  @include media(">=notebook") {
+    width: 50%;
+  }
+
   [data-breadcrumb-list] {
     display: flex;
     align-items: center;
+    margin-left: calc(-1 * var(--fs-spacing-1));
     overflow-x: hidden;
 
-    li {
+    [data-breadcrumb-list-item] {
```

Comment:
You can remove this from the cascade as it's now specified with a data-attribute

Reply:
I can't remove this from the cascade, because I use "first-child", so as Its rendered two breadcrumbs (mobile and desktop) generates bug